lib/Transformation.ts

```diff
@@ -197,3 +197,24 @@ export function transformAggregate(expr: Alg.AggregateExpression) {
 export function transformExistence(expr: Alg.ExistenceExpression) {
   return new E.Existence(expr);
 }
+
+// Ordering 2 given strings
+export function orderTypes(litA: RDF.NamedNode | RDF.BlankNode | RDF.Literal | RDF.Variable | RDF.DefaultGraph | undefined, 
+                        litB: RDF.NamedNode | RDF.BlankNode | RDF.Literal | RDF.Variable | RDF.DefaultGraph | undefined, 
+                        isAscending:boolean){
+  if (litA && litA.termType === "Literal" && litB && litB.termType === "Literal" ){
```

Comment:
We typically write our strings with single quotes (our linter is probably not configured strictly enough...).